### ISSUE TITLE
[Tensorflow 2]: Bring back tf.layers?

### ISSUE THREAD

**Issue by alanhdu** [tensorflow/tensorflow#26176]
In the proposed TensorFlow 2.0 API, `tf.layers` has been removed in favor of `tf.keras.layers`. Would it be possible to bring it back as a simple alias for `tf.keras.layers`? 

For reference, we already have:
- `tf.losses` == `tf.keras.losses`
- `tf.optimizers` == `tf.keras.optimizers`

as aliases. I personally think that `tf.keras.layers` are as-used as optimizers and loss functions, and that `tf.layers` would be a small ergonomic improvement over `tf.keras.layers`.

**Comment by MarkDaoust** [comment]
Hi @alanhdu, 

I think this is the plan. It's just not quite implemented yet.

Check again in a couple of weeks.

**Comment by jvishnuvardhan** [comment]
I am closing this as it was resolved. Thanks!

**Comment by alanhdu** [comment]
Hi @MarkDaoust! I've been playing around with the latest TF2 builds, and I noticed that `tf.layers` has not been exposed as an alias. Is there a tracking issue or something I could subscribe to know when the change happens if it's still planned?
